docs/clients.md

```diff
@@ -69,6 +69,42 @@ Therefore these two rules may help when choosing what to use:
 * When opening a client within a service, use local clients.  Usually you'll want to open one client per worker.
 * When opening a client from outside a service, such as from inside an actor or some other general use case, use a future client.
 
+## Failure Management
+ 
+### Client Behavior 
+ 
+If any failure occurs (example: connection closed, external system isn't 
+responding) the response is sent back as a failure and NOT retried.  The 
+default behavior is to continue trying subsequent requests.  If failfast is
+enabled then all queued requests in the ServiceClient are immediately removed.  
+
+### Retrying Requests
+
+To enable retries, the LoadBalancingClient (LBC) should be used.  The LBC takes
+in a generator that builds a ServiceClient from an InetSocketAddress and a list
+of InetSocketAddresses.  The LBC defaults to using the Integer max retry as the
```

Comment:
"Integer max retry" - that's true, but I don't think it tries 2,147,483,647 times. I think it tries as many times as there are connections, so if there are three connections it will try three times. @DanSimon am I correct?